src/main/resources/db/migration/common/V1_219__v48_without_transcription.sql

```diff
@@ -0,0 +1,17 @@
+-- V48 without transcription changes.
+ALTER TABLE court_case ALTER COLUMN case_number SET NOT NULL;
+ALTER TABLE court_case ALTER COLUMN case_closed SET NOT NULL;
+ALTER TABLE court_case ALTER COLUMN interpreter_used SET NOT NULL;
+
+ALTER TABLE node_register ALTER COLUMN device_type SET DEFAULT 'DAR';
+
+ALTER TABLE hearing ALTER COLUMN hearing_date SET NOT NULL;
+--ALTER TABLE hearing ALTER COLUMN scheduled_start_time SET NOT NULL;
```

Comment:
not sure about this change , I think in some cases like darts events we don't have a scheduled start time.

Reply:
Yeah there's lot of nulls there